### PR TITLE
fix imports to allow installation

### PR DESCRIPTION
**Pull request by ovnicraft**
Working to 7.0 geospatial i test it and need this fixes.

@nbessi please review

### CODE REVIEW

File: base_geoengine/__init__.py [hunk 1]
```diff
@@ -19,17 +19,17 @@
...
-from . import geo_db
-from . import geo_field
-from . import geo_point
-from . import geo_multipoint
-from . import geo_line
-from . import geo_multiline
-from . import geo_polygon
-from . import geo_multipolygon
-from . import company
+import geo_model
```
Comment (pedrobaeza) [inline base_geoengine/__init__.py:22]: Why are you removing this relative import?

**Comment by nbessi** [comment]
@ovnicraft that seems strange can you give me your version of Python?

**Comment by lmignon** [comment]
@ovnicraft Thank you for your PR.  However I have no problem when installing this module on a fresh instance of Odoo 7.0.  (tests done now) Your problems seem related to how your environment is configured. The only circumstances in which I have observed problems with relative imports in Odoo is when I tried to run the tests directly from my IDE using the Unittest2 nor Nose runner. If this problem occurs in this case, be aware that it is not possible to run the tests without going through the Odoo runner. Indeed, the Odoo patch the PYTHONPATH at startup to insert the addons under the module 'openerp.addons'.

**Comment by ovnicraft** [comment]
@nbessi 2.7.6 on osx
@lmignon i am working with virtualenv

**Comment by nbessi** [comment]
@ovnicraft that is quite strange. if you add a `from __future__ import absolute_import` did it change something. As far as I remember there was some oddities with OSX python with this

**Comment by ovnicraft** [comment]
@nbessi yes totally strange, btw i reproduce it in my vagrant machine (on OSX too).
Will test in my archlinux and give feedback.

**Pull request closed without merging**